front/lib/resources/conversation_resource.ts

```diff
@@ -493,6 +496,104 @@ export class ConversationResource extends BaseResource<ConversationModel> {
     );
   }
 
+  static async fetchMCPServerViews(
+    auth: Authenticator,
+    conversation: ConversationWithoutContentType,
+    onlyEnabled?: boolean
```

Comment:
`nit` to ease clarity at call sites.

```suggestion
    {onlyEnabled}: { onlyEnabled?: boolean}
```